metrics/web/lib/dashboard/presentation/widgets/build_number_scorecard.dart

```diff
@@ -23,12 +25,13 @@ class BuildNumberTextMetric extends StatelessWidget {
   Widget build(BuildContext context) {
     final widgetTheme = MetricsTheme.of(context).metricWidgetTheme;
 
-    if (buildNumberMetric == null || buildNumberMetric == 0) {
+    if (buildNumberMetric?.numberOfBuilds == null ||
+        buildNumberMetric?.numberOfBuilds == 0) {
```

Comment:
Should we display 0 when 0, not NoDataPlaceholder? I think we had some discussion before on that - but I just want to double-check.

Reply:
Will discuss this with Matthew on Monday.